### DISCUSSION TITLE
Sankey Dashboard

### DISCUSSION THREAD

**Discussion by Knot3n** [akvorado/akvorado#237] [Q&A]
Hi,

will or is it possible to have the SANKEY graph "Reversed"? 

Thanks for your great work by the way, really what you build here.

Kind Regards

Simon

**Comment by vincentbernat** [comment]
I don't understand what you mean. From top to bottom?

**Reply by Knot3n** [reply]
Hey, 

right now you see only the incoming traffic with SANKEY.

Example: https://github.com/akvorado/akvorado/raw/main/console/data/docs/sankey.png

My question is if its (or will) also possible to see the outgoing traffic towards the Internet with SANKEY. 

With the graph " stacked areas "  you can add bidirectional so see also the outgoing traffic, that option doesnt exist's with sankey.

**Reply by vincentbernat** [reply]
Change the filter to `OutIfBoundary = external`. I think that's what you need.

**Reply by Knot3n** [reply]
Yes, that is what i wanted. 

Thank you very much for your fast reply :)

**Reply by Knot3n** [reply]
Just a side question :

Do you know why when i save a filter and try to click on it - that it wont load?

**Reply by vincentbernat** [reply]
On demo.akvorado.net, I have added a saved filter. Are you able to load it?

**Reply by Knot3n** [reply]
yes it is working and i see my problem  .. so only the filter is getting saved but not the Dimensions.

Will be there a feature in the future to save also dimensions?

**Reply by vincentbernat** [reply]
At some point, there will be dashboards. In the meantime, feel free to bookmark the URL. It contains everything.
